src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/form/containers/GroupContainer.tsx

```diff
@@ -26,6 +26,20 @@ const gridStyle = {
   paddingTop: '12px',
 };
 
+const useStyles = makeStyles({
```

Comment:
This can be simplified a bit, if the result is not dynamic (f.ex adding/subtracting px values) there is no need to use `calc()` function.

```suggestion

const useStyles = makeStyles({
  minusMargin: {
    left: '24px',
    width: 'calc(100% + 48px)',
    position: 'relative',
    marginLeft: '-48px',
    '@media (min-width:768px)': {
      left: '37px',
      width: 'calc(100% + 154px)',
      marginLeft: '-74px',
    },
  }
});
```